clash-ghc/CHANGELOG.md

```diff
@@ -1,5 +1,149 @@
 # Changelog for the [`clash-ghc`](http://hackage.haskell.org/package/clash-ghc) package
 
+## 1.0.0 *September 3rd 2019*
+* New features:
+  * API changes: check the migration guide at the end of `Clash.Tutorial`
+  * All memory elements now have an (implicit) enable line; "Gated" clocks have
+    been removed as the clock wasn't actually gated, but implemented as an
+    enable line.
+  * Circuit domains are now configurable in:
+    * (old) The clock period
+    * (new) Clock edge on which memory elements latch their inputs
+      (rising edge or falling edge)
+    * (new) Whether the reset port of a memory element is level sensitive
+      (asynchronous reset) or edge sensitive (synchronous reset)
+    * (new) Whether the reset port of a memory element is active-high or
+      active-low (negated reset)
+    * (new) Whether memory element power on in a configurable/defined state
+      (common on FPGAs) or in an undefined state (ASICs)
+
+    * See the [blog post](https://clash-lang.org/blog/0005-synthesis-domain/) on this new feature
+  * Data types can now be given custom bit-representations: http://hackage.haskell.org/package/clash-prelude/docs/Clash-Annotations-BitRepresentation.html
+  * Annotate expressions with attributes that persist in the generated HDL,
+    e.g. synthesis directives: http://hackage.haskell.org/package/clash-prelude/docs/Clash-Annotations-SynthesisAttributes.html
+  * Control (System)Verilog module instance, and VHDL entity instantiation names
+    in generated code: http://hackage.haskell.org/package/clash-prelude/docs/Clash-Magic.html
```

Comment:
To consider for inclusion:

- Limited GADT support (pattern matching on vectors, or custom GADTs as longs as their usage can be statically removed)
- Experimental feature: multiple hidden clocks. Can be enabled by compiling Clash with `-fmultiple-hidden`.
- `-fclash-compile-ultra`
- Much improved infrastructure vfor handling of undefined (unknown) values
- Experimental feature: blackbox functions
-

Reply:
I don't want to mention compile-ultra